smoke-test/tests/browse/browse_test.py

```diff
@@ -51,7 +51,7 @@ def test_get_browse_paths(frontend_session, ingest_cleanup_data):
     # /prod -- There should be one entity
     get_browse_paths_json = {
         "query": get_browse_paths_query,
-        "variables": {"input": { "type": "DATASET", "path": ["prod"], "start": 0, "count": 10 } },
+        "variables": {"input": { "type": "DATASET", "path": ["prod"], "start": 0, "count": 100 } },
```

Comment:
Why this one?

Reply:
This fails when run our other environment where the fixtures can push the expected result past the 10th result.